docs/book/advanced-guide/getting-started-on-aws.md

```diff
@@ -0,0 +1,539 @@
+# Getting started with AWS
+
+This step-by-step guide explains how to set up and configure all the infrastructure necessary to run a ZenML pipeline on AWS.
```

Comment:
I would suggest extending this introduction with 1 or 2 sentences of motivation as this is one of the first guides toward a cloud deployment. We can also state this is an entry-level stack to AWS and further options exist if people would like to establish a more complicated stack.

Reply:
Copied/modified the intro from alexejs GCP guide to include some additional info.